x/bank/bench_test.go

```diff
@@ -15,12 +15,11 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	moduletestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
-	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/cosmos/cosmos-sdk/x/bank/testutil"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
-var moduleAccAddr = authtypes.NewModuleAddress(stakingtypes.BondedPoolName)
+var moduleAccAddr = types.NewModuleAddress(stakingtypes.BondedPoolName)
```

Comment:
So here, because we are in `bank` we should change everything to `authtypes` instead of just `types`.